### DISCUSSION TITLE
How to pass a buildx --secret to a Dockerfile?

### DISCUSSION THREAD

**Discussion by cardosov** [fabric8io/docker-maven-plugin#1741] [Q&A]
I have a `Dockerfile` that mounts a secret. Something like:
```
RUN --mount=type=secret,id=creds,target=/tmp/my.credentials ...
```
How can I do the equivalent of `docker build --secret id=creds,src=$HOME/creds/my-file .` through the plugin?

**Comment by rohanKanojia** [comment]
@cardosov : Are you referring to [Docker Build Secrets](https://docs.docker.com/build/building/secrets/)? It doesn't seem to be related only to buildx. I checked codebase and it doesn't seem like we support it at the moment.

**Comment by cardosov** [comment]
@rohanKanojia Thanks for your reply. Yes, I am referring to [Docker Build Secrets](https://docs.docker.com/build/building/secrets/).

Do you know of any plans to support it?
